imgui/src/io.rs

```diff
@@ -391,6 +396,30 @@ impl Io {
     pub fn update_delta_time(&mut self, delta: Duration) {
         self.delta_time = delta.as_secs_f32().max(f32::MIN_POSITIVE);
     }
+
+    pub fn add_mouse_pos_event(&mut self, pos: [f32; 2]) {
+        unsafe {
+            sys::ImGuiIO_AddMousePosEvent(self.raw_mut(), pos[0], pos[1]);
+        }
+    }
+
+    pub fn add_mouse_button_event(&mut self, button: MouseButton, down: bool) {
```

Comment:
Could the imgui-winit-support code be updated to use this and the `add_mouse_wheel_event` methods?

Reply:
Yes, will do that, it seems I just forgot updating anything but the keyboard events while testing :)